UnitTests/GitCommandsTests/Remote/GitRemoteManagerTests.cs

```diff
@@ -249,5 +249,105 @@ public void SetRemoteState_should_call_ToggleRemoteState(string remoteName, bool
             _configFile.Received(1).AddConfigSection(sections[remoteDisabled ? 1 : 0]);
             _configFile.Received(1).Save();
         }
+
+        [Test]
+        public void ConfigureRemotes_Should_not_update_localHead_if_remoteHead_is_local()
+        {
+            var refs = new List<IGitRef>
```

Comment:
You could use `new[] { ... }` instead of `new List<T> { ... }` here. The former will allocate less memory for two reasons:

- `Array` is smaller than `List<T>` (which internally uses an array plus some other stuff plus it's own object overhead)
- `List<T>` will allocate an array with some capacity (I believe it's 16 elements) so that future calls to `Add` don't need to grow the array, but in this case you're only adding one item.

Very minor, but you said you were interested in feedback.